internal/pkg/service/stream/definition/repository/schema/branch_schema.go

```diff
@@ -0,0 +1,37 @@
+package schema
+
```

Comment:
I have to create a new `schema` packages for direct usage in tests. 
The code is moved without modifications.